### PR TITLE
Style footer

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: layout/theme.liquid [hunk 1]
```diff
@@ -128,61 +130,96 @@
...
-            <a href="mailto:{{ settings.contact_email }}">{{ settings.contact_email }}</a>
-          </div>
-        {% endif %}
-        {% if settings.mailchimp_action != blank and settings.mailchimp_hidden != blank %}
-          <form action="{{ settings.mailchimp_action }}" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank">
-            <input placeholder="your email address" type="email" name="EMAIL" id="mce-EMAIL">
+          <nav class="grid-item item-s-12 item-m-auto no-gutter grid-row">
+            {% if collections.'men' != blank %}
+            {% assign collection = collections.'men' %}
+            {% include 'nav-collection' %}
```
Comment (varffs) [inline layout/theme.liquid:149]: is it weird that this triggers the submenu in the header?

**Review by varffs — COMMENTED** [review]
Just a question about UX with hovers on the footer collections. If you/anyone is happy with then then merge but I think maybe it is kinda wierd.

File: layout/theme.liquid (hunk 1 above)
Comment (mikeyrayvon) [reply layout/theme.liquid:149]: no it shouldnt trigger the submenu in the header

Comment (mikeyrayvon) [reply layout/theme.liquid:149]: fix it if you want

**Pull request merged**